spec/index.bs

```diff
@@ -27,6 +27,12 @@ Introduction {#intro}
 
 This specification rocks.
 
+Convention {#convention}
+========================
+
+## Coordinate Systems ## {#coordinate-systems}
+WebGPU's coordinate systems conform to DirectX and Metal's coordinate systems:
```

Comment:
I would say "match" instead of "conform to", since "conform" implies subservience.

Reply:
True.  Thank you, @jdashg . I updated accordingly, with some revision about z range in NDC